src/lib/ElectrumClient.js

```diff
@@ -394,11 +395,14 @@ export default class Client {
    *         hexadecimal form.
    */
   async getTransactionMerkle(txHash, blockHeight) {
-    return await this.electrumClient
-      .blockchain_transaction_getMerkle(txHash, blockHeight)
-      .catch(err => {
-        throw new Error(`failed to get transaction merkle: [${err}]`)
-      })
+    return backoffRetrier(3, _ => _.message.includes("not in block at height"))(
```

Comment:
The underscore syntax definitely wins points for being terse but if the error matcher function was written something like
```
err => err.message.includes("my message")
```
it might help with intent. 🤷‍♀️

Reply:
Yeah. Basically if it's one statement I always default to _, but agree that here the err would reallllyyyy help. Will make the change.